### PR TITLE
Add foreman_azure to config

### PR DESCRIPTION
**Pull request by dLobatog**
Depends on https://github.com/theforeman/puppet-foreman/pull/456

### CODE REVIEW

**Comment by domcleal** [comment]
Please also add a new config file migration, see ff04cd1 for an example.

File: config/foreman.migrations/20160615132930_foreman_azure.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
++answers['foreman::plugin::azure'] ||= false
```
Comment (domcleal) [inline config/foreman.migrations/20160615132930_foreman_azure.rb]: There's a leading `+` on these lines causing the migration (and tests) to fail.

**Comment by dLobatog** [comment]
Updated to fix the typo on the migration

**Comment by domcleal** [comment]
Merged, thanks @dLobatog!

**Pull request merged**